catroid/src/main/java/org/catrobat/catroid/ui/fragment/AddUserDataToUserDefinedBrickFragment.kt

```diff
@@ -0,0 +1,219 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2021 The Catrobat Team
```

Comment:
please update in every file
```suggestion
 * Copyright (C) 2010-2022 The Catrobat Team
```